dev/partkey.md

```diff
@@ -0,0 +1,165 @@
+---
+numbersections: true
+title: "Algorand Key Specification"
+date: \today
+abstract: >
+    This document specifies the list of keys and their capabilities in Algorand.
```

Comment:
General question:

How does batch/offset match round/steps?

Reply:
I think that I addresses that as well.
The voting key would be valid for the duration of a single round. It could sign multiple votes within that round.